pages/page.py

```diff
@@ -35,9 +34,7 @@ def __init__(self, testsetup):
     def is_the_current_page(self):
         if self._page_title:
             page_title = self.page_title
-            Assert.equal(page_title, self._page_title,
-                         "Expected page title: %s. Actual page title: %s" %
-                         (self._page_title, page_title))
+            assert page_title == self._page_title
```

Comment:
Changing to this syntax will present a detailed description of exactly what the mismatch is. So it if fails you will see both the expected value and the actual value, and the difference between them will be highlighted. I think that's even better than what we had before. This is one of the reasons we decided to move away from Unittestzero.